tasks/post-to-cmr/tests/cmr_test.js

```diff
@@ -31,56 +31,81 @@ test.afterEach.always(async (t) => {
   deleteBucket(t.context.bucket);
 });
 
-test.serial('should succeed if cmr correctly identifies the xml as invalid', (t) => {
+test.serial('should succeed if cmr correctly identifies the xml as invalid', async (t) => {
   sinon.stub(cmrjs.CMR.prototype, 'getToken');
 
   const newPayload = JSON.parse(JSON.stringify(payload));
   const granuleId = newPayload.input.granules[0].granuleId;
   const key = `${granuleId}.cmr.xml`;
 
-  return aws.promiseS3Upload({
-    Bucket: t.context.bucket,
-    Key: key,
-    Body: '<?xml version="1.0" encoding="UTF-8"?><results></results>'
-  }).then(() => postToCMR(newPayload)
-    .then(() => {
-      cmrjs.CMR.prototype.getToken.restore();
-      t.fail();
-    })
-    .catch((e) => {
-      cmrjs.CMR.prototype.getToken.restore();
-      t.true(e instanceof cmrjs.ValidationError);
-    }));
+  try {
+    await aws.promiseS3Upload({
+      Bucket: t.context.bucket,
+      Key: key,
+      Body: '<?xml version="1.0" encoding="UTF-8"?><results></results>'
+    });
+    const output = await postToCMR(newPayload);
```

Comment:
Extremely minor thing, but we don't need to save the output here since we're never referencing it.